hibernate-cache/src/test/java/org/infinispan/test/hibernate/cache/tm/JBossStandaloneJtaExampleTest.java

```diff
@@ -108,12 +108,10 @@ public void testPersistAndLoadUnderJta() throws Exception {
 			ut.begin();
 			try {
 				Session session = sessionFactory.openSession();
-				assertTrue(session.getTransaction().isActive());
+            session.getTransaction().begin();
```

Comment:
You should not work with Hibernate transactions explicitly - everything should be controlled from the JTA transaction object/through `j.t.TransactionManager`